db/migrate/20180806205420_add_index_to_content_view_version.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexToContentViewVersion < ActiveRecord::Migration[5.1]
+  def change
+    add_index :katello_content_view_versions, :major
+    add_index :katello_content_view_versions, :minor
```

Comment:
For this index, we need it to be unique across 3 columns,  content_view_id, major, and minor.  It'd look something like this:

add_index :katello_content_view_versions, [:content_view_id, :major, :minor], :unique => true

Reply:
I think we want to add a uniquness validation to the model as well, something like:
```
validates :minor :uniqueness => {:scope => [:content_view_id, :major]}
```